index.html

```diff
@@ -13,22 +13,30 @@
 <body>
     <!-- Toolbar -->
     <nav class="toolbar">
-        <button><img src="Assets/Icon - Menu.png" alt=""></b>
+        <button><img src="Assets/Icon - Menu.png" alt="hamburger"></button>
+        <ul>
+            <li> <a href="#portfolio">Portfolio</a> </li>
+            <li><a href="#about">About</a></li>
+            <li><a href="#contact">Contact</a></li>
+        </ul>
     </nav>
     <!-- Headline -->
     <section class="headline">
         <div class="headline_content">
             <div class="title">
-                <h1>
-                    Hey there. <br> I'm Ghaus. <br> <span>I'm a software <br> developer</span>
-                </h1>
+                <div>
+                    <h3>Hey there.</h3>
+                    <h3>I'm Ghaus.</h3>
+                </div>
+                <h3><span>I'm a software developer</span></h3>
             </div>
             <div class="description">
                 <p>
                     I can help you build a product , feature or website Look through some of my work and experience! If
```

Comment:
Please kindly adjust your background image for the `headline` section so it contains the full page here instead of being clustered around the text just as it is in the Figma template given

Your design :
![image](https://github.com/GhausAliKhan/My-Portfolio/assets/20537712/ecdf7fae-717f-48b0-bf8d-7cb94a2a4db2)

Figma design :
![image](https://github.com/GhausAliKhan/My-Portfolio/assets/20537712/f2438637-9d4b-4f43-ac3e-597c1f9bda9e)